entity-framework/toc.yml

```diff
@@ -219,6 +193,32 @@
         href: core/saving/disconnected-entities.md
       - name: Explicit values for generated properties
         href: core/saving/explicit-values-generated-properties.md
+    - name: Miscellaneous
```

Comment:
Not a fan of the title *Miscellaneous*, but I like the grouping.

Reply:
We should discuss as a team how to make this work with Interception, Diagnostics, Logging and Change tracking topics as well

Reply:
Two irrefutable arguments in favor of Miscellaneous:

1. The URL slug is already this way
2. The person doing the PostgreSQL provider decided to use Miscellaneous [in their docs](http://www.npgsql.org/efcore/index.html) so it must be correct

More seriously though :sweat_smile: It's really just about the topics inside being mixed/not intrinsically related to one another ([definition](https://www.dictionary.com/browse/miscellaneous?s=t)).

But yeah, let's discuss all this in design.